examples/core-example-advanced/src/state/actions/selection/paste.ts

```diff
@@ -0,0 +1,13 @@
+import { nanoid } from 'nanoid'
+import type { Action } from 'state/constants'
+
+export const paste: Action = (data) => {
+  const selectedShapes = Object.values(data.page.shapes).filter((shape) => shape.isCopied)
+
+  const newShapes = selectedShapes.map((s) => ({ ...s, id: nanoid() }))
```

Comment:
you can do it like this
```ts
const newShapes = selectedShapes.map((s) => ({
    ...s,
    id: nanoid(),
    point: [s.point[0] + 10, s.point[1] + 10],
  }))```

Reply:
That's a lovely addition @judicaelandria , added now.